test/integration/integration_test.go

```diff
@@ -279,31 +288,57 @@ func checkConfigKey(configPath string, condition string, keyPath string) error {
 
 func compareExpectedWithActualContains(expected string, actual string) error {
 	if !strings.Contains(actual, expected) {
-		return fmt.Errorf("Output did not match. Expected: %s, Actual: %s", expected, actual)
+		return fmt.Errorf("Output did not match. Expected: '%s', Actual: '%s'", expected, actual)
 	}
 
 	return nil
 }
 
 func compareExpectedWithActualNotContains(notexpected string, actual string) error {
 	if strings.Contains(actual, notexpected) {
-		return fmt.Errorf("Output did match. Not expected: %s, Actual: %s", notexpected, actual)
+		return fmt.Errorf("Output did match. Not expected: '%s', Actual: '%s'", notexpected, actual)
 	}
 
 	return nil
 }
 
 func compareExpectedWithActualEquals(expected string, actual string) error {
 	if actual != expected {
-		return fmt.Errorf("Output did not match. Expected: %s, Actual: %s", expected, actual)
+		return fmt.Errorf("Output did not match. Expected: '%s', Actual: '%s'", expected, actual)
 	}
 
 	return nil
 }
 
 func compareExpectedWithActualNotEquals(notexpected string, actual string) error {
 	if actual == notexpected {
-		return fmt.Errorf("Output did match. Not expected: %s, Actual: %s", notexpected, actual)
+		return fmt.Errorf("Output did match. Not expected: '%s', Actual: '%s'", notexpected, actual)
+	}
+
+	return nil
+}
+
+func compareExpectedWithActualMatches(expected string, actual string) error {
+	expectedRegexp, err := regexp.Compile(expected)
+	if err != nil {
+		return fmt.Errorf("Expected value must be a valid regular expression statement: ", err)
+	}
+
+	if !expectedRegexp.MatchString(actual) {
+		return fmt.Errorf("Output did not match. Expected: '%s', Actual: '%s'", expected, actual)
+	}
+
+	return nil
+}
+
+func compareExpectedWithActualNotMatches(notexpected string, actual string) error {
```

Comment:
@gbraad Thank for feedback, going to implement it. Also thinking if I can take code from regexp matching for config files, will try it.